app/models/gsa18f/proposal_form.rb

```diff
@@ -0,0 +1,105 @@
+#module Ncr
+module Gsa18f
+  DATA = YAML.load_file("#{Rails.root}/config/data/18f.yaml")
+
+  class ProposalForm
+    include SimpleFormObject
+
+    URGENCY = DATA['URGENCY']
+    OFFICES = DATA['OFFICES']
+    
+    #18f additions
+    #attribute :approver_email, :text
+    attribute :requester, :user
+    attribute :office, :string
+    attribute :justification, :text
+    attribute :link_to_product, :string
+    attribute :quantity, :decimal
+    attribute :date_requested, :string
+    attribute :urgency, :string
+    attribute :additional_info, :string
+    attribute :cost_per_unit, :decimal
+    attribute :product_name_and_description, :text
+    attribute :origin, :string
+
+    validates :cost_per_unit, numericality: {
+      greater_than_or_equal_to: 0,
+      less_than_or_equal_to: 3000
+    }
+    validates :product_name_and_description, presence: true
+
+    # def budget_approver_email
+    #   ENV['NCR_BUDGET_APPROVER_EMAIL'] || 'communicart.budget.approver@gmail.com'
+    # end
+
+    # def finance_approver_email
+    #   ENV['NCR_FINANCE_APPROVER_EMAIL'] || 'communicart.ofm.approver@gmail.com'
+    # end
+
+    # def requires_finance_approval?
+    #   self.expense_type == 'BA61'
+    # end
+
+    def approver_emails
+      emails = ['Richard.L.Miller@gsa.gov']
+      emails
+    end
+
+    def create_cart
+      cart = Cart.new(
+        name: self.product_name_and_description,
+        proposal_attributes: {flow: 'linear'}
+      )
+      if cart.save
+        self.set_props_on(cart)
+        self.approver_emails.each do |email|
+          cart.add_approver(email)
+        end
+        Dispatcher.deliver_new_cart_emails(cart)
+      end
+
+      cart
+    end
+
+    def update_cart(cart)
+      cart.name = self.product_name_and_description
+      if cart.save
+        cart.approver_approvals.destroy_all
```

Comment:
On NCR it's required as the user could select different addies when restarting the form. That's less a concern here (as the approvers are constant)